app/connectors/VatRegistrationConnector.scala

```diff
@@ -93,17 +93,17 @@ trait RegistrationConnector {
   }
 
   def deleteVatScheme(regId: String)
-                     (implicit hc: HeaderCarrier, rds: HttpReads[Boolean]): Future[Boolean] = {
+                     (implicit hc: HeaderCarrier, rds: HttpReads[Boolean]): Future[Unit] = {
```

Comment:
We will raise a story to re-visit this - so that we are actually returning something that tells us what has happened, as opposed to a Unit or a Boolean.